src/ansys/sound/core/psychoacoustics/spectral_centroid.py

```diff
@@ -114,7 +114,12 @@ def get_output_as_nparray(self) -> npt.ArrayLike:
         numpy.ndarray:
             Array of spectral centroid values in Hz.
         """
-        return np.array([self._output])
+        output = self.get_output()
+
+        if output == None:
+            return None
```

Comment:
@anshlachamb As I tried to explain we should return np.array([]) instead of None here. This way the returned type is still consistent with the method prototype. Unless there is a particular reason against that...

Reply:
done